README.md

```diff
@@ -10,8 +10,8 @@ Users with a Google Account can also store past, current or future schedules of
 
 | Type | Link       | 
 |------|------------|
-| prod | <https://proj-courses.dokku-00.cs.ucsb.edu/> | 
-| qa | <https://proj-courses-qa.dokku-00.cs.ucsb.edu/>  | 
+| prod | <https://proj-courses.dokku-12.cs.ucsb.edu/> | 
+| qa | <https://proj-courses-qa.dokku-12.cs.ucsb.edu/>  | 
 
```

Comment:
This ought to be in it's own PR for 5 points... if you include it in this one, you are cheating yourself and your team out of 5 points.